### DISCUSSION TITLE
Cs50 course and the course

### DISCUSSION THREAD

**Discussion by engaboud** [community/community#139080] [Other Feature Feedback, Questions, & Ideas] [answered]
Greetings to you

I have completed the Cs50 course and the course appears to me as completed but the accredited certificate does not appear or I have steps left to complete which I have not been able to complete for a month despite all my attempts. We hope to help me reach the one I attended in 2023 and also in 2024 and I did not receive my certificate and I paid all the course fees

**Comment by SavinRazvan** [comment]
Try this and see if you made your submissions correct:
https://submit.cs50.io/users/your-username-here

**Comment by ebndev** [comment, marked as answer]
Hi @engaboud,

Thanks for being a part of the GitHub Community, we're glad you're here!

You might want to try asking checking the [CS50 FAQ](https://cs50.harvard.edu/college/2024/fall/faqs/#:~:text=Email%20heads@cs50.harvard.,Prior%20Experience) for answers or to reach out to them using the contact information on the page. 

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
